README.md

```diff
@@ -191,12 +191,16 @@ By default toasts are not shown.
     - TODO([#39](https://github.com/jackbsteinberg/std-toast/issues/39)): Do we need values `"top-stretch"`, `"center-stretch"`, and `"bottom-stretch"` as well? Should this stretching be done automatically on mobile?
 The default (if the attribute is omitted or set to an invalid value) is ???.
     - TODO([#13](https://github.com/jackbsteinberg/std-toast/issues/13)): should this positioning be an attribute or a style
-- `closebutton`: a boolean attribute, determining whether an explicit close button is shown. 
-By default toasts do not have a close button.
-See ["Contents"](#contents) and ["Appearance customization"](#appearance-customization) sections for how to customize the close button when it's shown.
+- `closebutton`: allows setting the toast's close button content (e.g. `<std-toast closebutton="Dismiss">`),
+or leaving it up to the user agent (e.g. `<std-toast closebutton>`).
```

Comment:
Should we expand the "leaving it up to the user agent" phrase to explain there's default closebutton content that will show up? Also, what does "leaving it up to the user agent" mean here? Is that leaving it for the browser to decide? JS to fill it in?

Reply:
Up to the browser (= user agent) to decide. Will try to tweak.